source/administration/devops-command-center.rst

```diff
@@ -64,48 +64,122 @@ Viewing and creating playbooks
 
 Select **!** in the channel header to open the RHS panel. Select the **Playbooks** icon to open the playbooks backstage and list all playbooks associated with the current team. Select **+ New Playbook** to start a new playbook and checklist. Enter a playbook name and choose **Save Playbook**.
 
-Creating checklists
-~~~~~~~~~~~~~~~~~~~
+Public and private incidents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Checklists are created within a playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a playbook, or during an incident.
+Incident channels can be public or private, based on the settings of the playbook used for the incident.
 
-Editing checklists
-~~~~~~~~~~~~~~~~~~~
+- **Public:** Anyone in the team can view the incident details and join the incident channel. The incident details are visible to all team members during the active incident and after the incident has ended.
+- **Private:** Only incident members who were members of the channel when the incident started have access to the ongoing incident and incident details after the incident has ended. Private incidents can't be made public once they've been started.
+
+Private incidents don't appear in the RHS or in the backstage when they're active, and the contents of the channel won't show up in a search query. If an incident was public and changed to private, it can't be accessed by non-participants unless they're invited by a member of the incident channel.
```

Comment:
```suggestion
Private incidents don't appear in the backstage when they're active, and the contents of the channel won't show up in a search query. If an incident was public and changed to private, it can't be accessed by non-participants unless they're invited by a member of the incident channel.
```